cfgov/v1/wagtail_hooks.py

```diff
@@ -49,6 +49,17 @@ def share(page, is_sharing, is_live):
     page.save()
 
 
+@hooks.register('insert_editor_js')
```

Comment:
Can you please use an indentation level of 4 spaces here to be consistent with the rest of the code and [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation)?